### PR TITLE
Add emotion theming to application

### PR DESCRIPTION
**Pull request by jleach**
This PR is meant as a way to do a preliminary review of the emotion theming pattern.

### CODE REVIEW

File: App/components/inputs/TextInput.tsx [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
+  display: flex;
+  flex-grow: 1;
+  flex-direct: column;
+`
+const InputLabel = styled.Text`
+  font-size: 16px;
+  margin: 2px;
+  color: ${(props) => props.theme.colors.mainColor};
+`
+const PinEntry = styled.TextInput`
```
Comment (amanji) [inline App/components/inputs/TextInput.tsx:20]: Wondering if we could generalize these even further into global base styliz(ed|able) components

File: App/screens/PinCreate.tsx [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
-
-import { Button, TextInput, SafeAreaScrollView } from 'components'
-
-interface Props {
-  route: RouteProp<AuthenticateStackParams, 'Create 6-Digit Pin'>
+interface Props extends TextInputProps {
+  label: string
 }
 
+const Row = styled.View`
```
Comment (amanji) [inline App/screens/PinCreate.tsx:11]: Similar to the comment above, perhaps we could have a global set of flex, grid and container classes

**Review by amanji — APPROVED** [review]

File: App/screens/PinCreate.tsx (hunk 2 above)
Comment (jleach) [reply App/screens/PinCreate.tsx:11]: Yes. Over time. This PR is to just start the conversation with the broader community on the pattern to theme the app.

File: App/components/inputs/TextInput.tsx (hunk 1 above)
Comment (jleach) [reply App/components/inputs/TextInput.tsx:20]: Have an example? Not sure what it means.

**Comment by jleach** [comment]
Closing. Will re-create a PR to the Hyperledger org.

**Pull request closed without merging**